pkg/omf/functions/cli/omf.cli.describe.fish

```diff
@@ -1,4 +1,12 @@
-function omf.cli.describe -a name
-  omf.packages.describe $name
-  return 0
+function omf.cli.describe -a package -d 'Show information about a package'
+  if set -l props (omf.index.stat $package description repository maintainer)
+    echo "Package: $package"
+    echo "Description: $props[1]"
+    echo "Repository: $props[2]"
+    echo "Maintainer: $props[3]"
```

Comment:
What happens if one of the keys are missing in the index entry?
I wouldn't bother with it now, because for now we're the only ones maintaining an index, and it's not so hard to keep a few keys for each package.
Just saying... builtin associative arrays in fish would fix this in a heartbeat (and will probably let some PowerShell ugliness into our beloved fish).